lib/Doctrine/DBAL/Types/DateIntervalType.php

```diff
@@ -22,7 +22,7 @@ public function getName()
      */
     public function getSQLDeclaration(array $fieldDeclaration, AbstractPlatform $platform)
     {
-        $fieldDeclaration['length'] = 20;
+        $fieldDeclaration['length'] = 255;
```

Comment:
Yeah, but theoretically you can add as many numbers in any interval part as you want, so i.e. MySQL 64, 128 or other length lower than 255 (varchar) reservers same amount of memory, so is it really needed to calculate potential minimum?